ResourceServer/Controllers/UserController.cs

```diff
@@ -28,32 +24,28 @@ public class UserController : ControllerBase
             _configuration = configuration;
             _logger = logger;
         }
-        // Post: api/User
-        /**
-         * gets information about user
-         * **/
-        [HttpPost]
-        public string Get(string username)
+
+        [HttpGet("phoneNumber")]
+        public JObject getPhoneNumber()
         {
-            if(username == null)
-            {
-                return "Missing username in body";
-            }
-            User user = TrueHomeContext.getUserByLogin(username);
-            if(user != null)
-            {
-                return JsonConvert.SerializeObject(user, Formatting.Indented);
-            }
-            return "Missing user with this name";
+            var userId = User.FindFirst("sub")?.Value;
+
+            string phoneNumber = TrueHomeContext.getPhoneNumber(userId);
+            string jsonData = null;
+            if (phoneNumber == null)
+                jsonData = "{ 'phoneNumber': null }";
+            else
+                jsonData = "{'phoneNumber': " + $"'{phoneNumber}'" + "'}";
```

Comment:
Czy JSON dobrze sie parsuje przy znakach ' ? w JOSNie stringi muszą być między "cudzysłowami" więć nwm czy nie powinno być \" zamiast ' ale to wyjdzie przy testach